src/main/java/com/softserveinc/ita/steps/SpecialitiesSteps.java

```diff
@@ -22,9 +23,10 @@ public void openSpecialityPage() {
     public void addNewSpeciality(SpecialityEntity speciality) {
         page
                 .openAddingNewForm()
-                .setCode(speciality.getCode())
-                .setName(speciality.getName())
-                .confirmModal()
+                .setValueFor(SPECIALTY_CODE, speciality.getCode())
+                .setValueFor(SPECIALTY_NAME, speciality.getName())
+                .confirmModal();
+        page
                 .waitForProgressBarToAppear()
```

Comment:
you could create some method and merge this methods:

```
waitTillProgressbarDisappear(){
waitForProgressBarToAppear()
.waitForProgressBarToDisappear();
}
```

and to use in confirmModal before return statement